### PR TITLE
test: update to match new zoe/tools/manualTimer.js

### PR DESCRIPTION
**Pull request by warner**
In https://github.com/Agoric/agoric-sdk/pull/5847 , Zoe's
`tools/manualTimer.js` is updated to align with the new SwingSet
TimerService. In this new version, when the ManualTimer is configured
with `eventLoopIteration`, an `await tick()` will properly drain the
promise queue before proceeding. In addition, the TimerService will
fire a `setWakeup(now)` immediately, rather than requiring a `tick()`
first.

This changes the behavior of tests which were not correctly
synchronizing before, especially the timer-based
fakePriceAuthority. Where previously you could set a price sequence of
`[20, 55]` and use a single `await tick()` to get to `time=1` and
`price=20`, in the new version, you use no ticks, and start out with
`time=0` and `price=20`. A single `await tick()` gets you `time=1` and
`price=55`.

This requires changes to the unit tests, in general removing one
`tick()` and decrementing the expected timestamp by one.

### CODE REVIEW

**Comment by warner** [comment]
This should not land until after the new vat-timer code lands, but *should* land shortly thereafter.

**Comment by warner** [comment]
@michaelfig the new vat-timer has landed on trunk, so this wants to get merged so we can un-break the agoric-sdk CI test for everyone

**Review by turadg — APPROVED** [review]
LGTM except the new import. Discussion started at https://github.com/Agoric/agoric-sdk/pull/5847/files#r950239426

**Pull request merged**
